com.avaloq.tools.ddk.checkcfg.ui/src/com/avaloq/tools/ddk/checkcfg/ui/CheckCfgUiModule.java

```diff
@@ -35,7 +36,7 @@ public CheckCfgUiModule(final AbstractUIPlugin plugin) {
   }
 
   /**
-   * Binds a {@link XtextTemplateContextType} which adds {@link SimpleEnumTemplateVariableResolver}.
+   * Binds a {@link XtextTemplateContextType} which adds {@link ResourceNameTemplateVariableResolver} and {@link SimpleEnumTemplateVariableResolver}.

```

Comment:
I'm a really big fan of not doing an import just for javadoc links. If you want `ResourceNameTemplateVariableResolver` and `SimpleEnumTemplateVariableResolver` linked in the javadoc, please use the fully-qualified names and get rid of the imports.

Reply:
Changed to fully-qualified names.